src/components/shared/forms/SearchDropdown.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <semantic-form-dropdown value="" :options="[]" :settings="computedSettings" :placeholder="placeholder" search/>
+    <semantic-form-dropdown value="" :options="[]" :settings="computedSettings" :placeholder="placeholder" search :fluid="fluid"/>
```

Comment:
Instead of declaring and then handling individual props, you can use `v-bind="$attrs"` to transparently pass through all attributes that aren't declared as props in the parent component.
 
```html
<semantic-form-dropdown value="" :options="[]" :settings="computedSettings"` v-bind="$attrs" />
```

See this [fiddle](https://jsfiddle.net/eywraw8t/136520/)

This decouples the parent and child component and means that we won't have to add further props in the future if we want to expose more of the child's functionality.

Reply:
Thanks for pointing that out and the demo, i have made the change and will take note.